macaroonbakery/oven.py

```diff
@@ -60,7 +62,7 @@ def __init__(self, key=None, location=None, locator=None, ns=None,
             my_store = store.MemoryKeyStore()
             self.root_keystore_for_ops = lambda x: my_store
 
-    def macaroon(self, version, expiry, caveats, ops):
+    def macaroon(self, ctx, version, expiry, caveats, ops):
```

Comment:
Please document argument and return types. Also, I'm not sure that we should be passing context around as freely as we do in Go - in pybakery, the context is tied more strongly to auth, but in Go it's used for network operations too.

In this case, I don't think that the context is being used for auth purposes, so should probably be omitted.
That probably applies to other Oven methods too.